### PR TITLE
SC-383: Fixes

### PR DESCRIPTION
**Pull request by xschildw**
This PR puts back the proxypassreverse fix from #75. We've had intermittent problem without it (i.e. sometimes we don't see the redirects and it works, other times they are there and it fails... Not clear what causes the difference in behavior). 
Also installing the synapser package as the ubuntu user (else the user gets prompted when first using the package).

### CODE REVIEW

File: src/proxy.conf [hunk 1]
```diff
@@ -12,11 +12,11 @@
   RewriteCond %{HTTP:Upgrade} !=websocket
   RewriteRule /EC2_INSTANCE_ID/(.*)     http://localhost:8787/$1 [P,L]
 
-  <LocationMatch /EC2_INSTANCE_ID/ >
+  <Location /EC2_INSTANCE_ID/ >
```
Comment (zaro0508) [inline src/proxy.conf:15]: i'm fine with this but FWIW, `LocationMatch` worked for me.

File: src/playbook.yaml [hunk 2]
```diff
@@ -109,6 +105,7 @@
 
     # Install essential R packages
     - name: Install synapser
+      become_user: ubuntu
```
Comment (zaro0508) [inline src/playbook.yaml:108]: questions..
1. are you sure ubuntu user has enough privileges to install R packages?
2. why become ubuntu user only for synapser package?  why not be consistent and install all R packages with the same user?

File: src/playbook.yaml [hunk 3]
```diff
@@ -94,10 +94,6 @@
           echo 'LoadModule python_module /usr/lib/apache2/modules/mod_python.so' >> /etc/apache2/mods-available/python.load
         chdir: /tmp/mod_python_source
 
-    # Install python synapse client
-    - name: Install Python Synapse client
```
Comment (zaro0508) [inline src/playbook.yaml:98]: could you explain why you are removing this now?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: src/playbook.yaml (hunk 3 above)
Comment (xschildw) [reply src/playbook.yaml:98]: The synapseclient is installed in a venv by reticulate/synapser.

File: src/playbook.yaml (hunk 2 above)
Comment (xschildw) [reply src/playbook.yaml:108]: 1. I was able to install a package as ubuntu
2. Good point, changing that...

File: src/proxy.conf (hunk 1 above)
Comment (xschildw) [reply src/proxy.conf:15]: Let's just go with the original fix as-is...

**Pull request merged**